addons/addons.json

```diff
@@ -80,4 +80,6 @@
   "folders",
   "message-filters",
   "block-palette-icons"
```

Comment:
missing comma

Reply:
Apparently the merge did something weird. This addon should be at the end of the list.